converter/SEMSoutput.py

```diff
@@ -140,29 +174,69 @@ def add_entry(precinct_id, contest_id, answer):
         precinct_id = cvr_obj["_precinctId"]
 
         for contest in contests:
+            if contest['type'] == 'ms-either-neither':
+                either_neither_contest_id = contest["eitherNeitherContestId"]
+                pick_one_contest_id = contest["pickOneContestId"]
+
+                either_neither_answer = cvr_obj.get(either_neither_contest_id, None)
+                pick_one_answer = cvr_obj.get(pick_one_contest_id, None)
+
+                # both of these should be present, otherwise it's an invalid CVR and we shouldn't count this contest at all
```

Comment:
Is this true? I thought you could pick "neither" and leave the pick-one options blank.

Reply:
Ah, I guess "blank" would be an empty array?

Reply:
present as in it should be at least `[]`. `[]` means no user selection.

Reply:
I have a ticket assigned to you I think about making sure the scanner always outputs `contest_id: []` for blank contests that are on the page, which I think you're already doing.

Reply:
It should already be doing that, yeah.